packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx

```diff
@@ -61,8 +93,65 @@ export class GuxAvatar {
     }
   }
 
-  componentWillLoad(): void {
+  private renderBadge(): JSX.Element | null {
+    if (this.hasNotifications) {
+      return this.renderNotificationsBadge();
+    } else if (this.hasBadge) {
+      return (
+        <div
+          class={{
+            'gux-avatar-badge': true,
+            [`${this.presence}`]: true,
+            [`gux-${this.size}`]: true
+          }}
+        >
+          <gux-icon
+            icon-name={this.getPresenceIcon(this.presence)}
+            screenreader-text={this.i18n(this.presence)}
```

Comment:
I don't think we can do localisation like this. We have to allow the app to pass in the localised presence name. Maybe if they don't supply one we can fallback to this.